auto.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/expect -f
```

Comment:
Czemu tak?

Reply:
Faktycznie flaga -f była niepotrzebna. Poza tym dzięki tej komendzie mam dostęp do komend expected.

Reply:
Chodziło mi o wybór `expect`. Nie, że jest zły, jest całkiem w dechę, ale to nie do końca jest Bash. Spójrz:

```
[tammo@lafk-T470 ~]$ man expect                                                                                                                                                                           [INSERT]
Brak podręcznika dla expect
[tammo@lafk-T470 ~]$ expect                                                                                                                                                                               [INSERT]

Command 'expect' not found, but can be installed with:

sudo snap install expect  # version 5.45-7snap0, or
sudo apt  install expect

See 'snap info expect' for additional versions.
```

Reply:
Mogę go zainstalować, ale sam fakt, że potrzebuję, znaczy że nie jest on domyślnie obecny. Czy zatem np. nie chcesz zaktualizować prerekwizytów w pliku z instrukcjami?